onnxruntime/core/graph/graph_utils.cc

```diff
@@ -216,6 +226,20 @@ static bool RemoveNodeWithSingleNodeInSingleUsedOutput(Graph& graph, Node& node)
   return true;
 }
 
+/** Move the input edges that src_node has to target_node.
+After the move is complete src_node will have no input edges.
+*/
+static void MoveAllNodeInputEdges(Graph& graph, Node& src_node, Node& target_node) {
```

Comment:
Do you need a check that the two nodes have the same number of input args? Or that target has at least as many as the src?

Reply:
I'm leaving the check out of here and assuming the individual fusions will do that. I don't think you know for sure what is correct because you'd need to know about whether any optional inputs are allowed/expected for either node.